### ISSUE TITLE
Working sliding windows with sound change

### ISSUE THREAD

**Issue by CptChaos0815** [flybywiresim/a32nx#4589]
**Is your feature request related to a problem? Please describe.**
N/A

**Describe the feature you'd like to see implemented**
Sliding windows would be nice to have. 

**References**
https://www.youtube.com/watch?v=BTRMAKTS8wE

**Additional context**
The sound engineers could make some magic with the opening and closing sound and the changing engine/PTU sounds when the window is open.

Discord username (if different from GitHub):

**Comment by hotshotp** [comment]
You bet I'll make it so opening the window changes the sound, it will be very nice and realistic once that is implemented 😍

**Comment by XpNvH** [comment]
yes this needs to be added because it is in the fenix but i want to fly neo

**Comment by Benjozork** [comment]
> yes this needs to be added because it is in the fenix but i want to fly neo

This won't happen until the custom model.
